tests/by-util/test_date.rs

```diff
@@ -334,6 +334,29 @@ fn test_invalid_format_string() {
     assert!(result.stderr_str().starts_with("date: invalid format "));
 }
 
+#[test]
+fn test_date_string_human() {
+    let date_formats = vec![
+        "1 year ago",
+        "1 year",
+        "2 months ago",
+        "15 days ago",
+        "1 week ago",
+        "5 hours ago",
+        "30 minutes ago",
+        "10 seconds",
+    ];
+    let re = Regex::new(r"^\d{4}-\d{2}-\d{2} \d{2}:\d{2}\n$").unwrap();
+    for date_format in date_formats {
+        new_ucmd!()
+            .arg("-d")
+            .arg(date_format)
+            .arg("+%Y-%m-%d %S:%M")
```

Comment:
I guess this is a typo:
```suggestion
            .arg("+%Y-%m-%d %H:%M")
```

Reply:
clearly, thanks :)